pkg/reconciler/openshift/common/cabundle.go

```diff
@@ -114,10 +114,8 @@ func ApplyCABundles(u *unstructured.Unstructured) error {
 			}
 		}
 
-		// /etc/ssl/certs is the default place where CA certs reside in *nix
-		// however this can be overridden using SSL_CERT_DIR, let's check for
-		// that here.
-		sslCertDir := "/etc/ssl/certs"
+		// we wiil mount the certs at this location so we don't override the existing certs
+		sslCertDir := "/custom-certs"
```

Comment:
nit: 
```suggestion
		sslCertDir := "/tekton-custom-certs"
```
this will give be helpful in indicating to someone who's unaware what's inside this dir

Reply:
resolved